.github/CONTRIBUTION.md

```diff
@@ -19,11 +19,25 @@ Run lint with:
 
 `yarn lint`
 
-## Publishing
+## Releasing
 
-Currently only admins are able to publish.
+Currently only admins are able to create a release. A release consists of the following:
 
-1. Check out master and make sure it is up to date
-2. Run `npm version minor/patch`
-3. Verify changes in `package.json`, commit and run `git push && git push --tags`
-4. Make sure all checks pass, then Circle automatically publishes to NPM.
+- Bumping the package version based on your commits
+- Updating the API specification
+- Creating a new commit with the changed files
+- Creating a tag with the new version
+- Pushing the release commit and tag to master
+
+### Step-By-Step
+
+1. Check out master and run `git pull`.
+1. Run `git clean -dfx && yarn` to make sure depenencies are up-to-date.
```

Comment:
Looks like the numbers are not getting incremented

Reply:
No need to, that is a way of creating a numbered list with markdown. It is easier if you would add a step inbetween already existing steps for example.

Reply:
Cool! Did not know 👍